### ISSUE TITLE
NVDA and aria-polite attribute in Firefox.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4857]
**Reported by khsruru on 2015-01-28 15:40**
Hello, I’m Hyongsop Kim in Korea and NVDA user.
In Firefox, aria-live=”polite” live region doesn’t work.
Here is a sample page.
In that page, once I press close button, NVDA should say ‘region role is removed’ because this is the aria-polite, but nothing.
However in Internet Explorer, NVDA says correctly.
Please test with that page and fix the issue.
Thank you so much.

**Comment by leonardder** [comment]
@khsbory: Could you provide an update for this issue? Does it still occur?

**Comment by Adriani90** [comment]
I am closing this issue because we did not hear from the author for a very long time and we do not have a test website.
